flexget/components/managed_lists/lists/pending_list/api.py

```diff
@@ -264,6 +278,41 @@ def post(self, list_id, session=None):
         return response
 
 
+@pending_list_api.route('/<int:list_id>/entries/batch')
+@api.doc(params={'list_id': 'ID of the list'})
+@api.response(NotFoundError)
+class PendingListEntriesBatchAPI(APIResource):
+    @api.response(201, model=pending_lists_entries_return_schema)
+    @api.response(204)
+    @api.validate(model=pending_list_batch_schema)
+    @api.doc(description='Approve, reject, or remove multiple entries')
+    def post(self, list_id, session=None):
+        """Perform operations on multiple entries"""
+        data = request.json
+        entry_ids = data.get('ids')
+        operation = data.get('operation')
+        try:
+            entries = db.get_entries_by_list_id(list_id, entry_ids=entry_ids, session=session)
+        except NoResultFound:
+            raise NotFoundError('could not find entries in list %d' % list_id)
+
+        response = None
+        if operation == 'remove':
```

Comment:
i think it'd be nicer to separate this endpoint to two; one for delete (using the DELETE verb) and the other for approve/reject via PUT, in a similar fashion like we do for a single entry.

Reply:
DELETE can't have a body so it makes it kind of annoying semantically...but I can split them regardless.

Reply:
Firs hit from google

"The latest update to the HTTP 1.1 specification (RFC 7231) explicitly permits an entity body in a DELETE request:"

Reply:
My bad, i misread some stack overflow post that said otherwise.

Reply:
done.